lib/StaticNarrative/config.py

```diff
@@ -0,0 +1,51 @@
+"""Config for the StaticNarrative app."""
+
+import os
+from typing import Any
+from urllib.parse import urlparse
+
+
+def generate_config(config: dict[str, Any] | None) -> None | dict[str, Any]:
+    """Generate the config for the StaticNarrative, with some tweaks."""
+    if not config:
+        return {}
+
+    kbase_endpoint = config.get("kbase-endpoint")
+    if kbase_endpoint == "{{ kbase_endpoint }}":
+        msg = "Config file has not been populated correctly"
+        raise RuntimeError(msg)
```

Comment:
this would only happen if the config file had not been populated by `prepare_deploy_config.py`